demisto_sdk/commands/init/initiator.py

```diff
@@ -468,7 +468,11 @@ def yml_reformatting(self, current_suffix: str, integration: bool = False):
         yml_dict["commonfields"]["id"] = self.id
         yml_dict['name'] = self.id
 
-        if LooseVersion(yml_dict.get('fromversion', '0.0.0')) < LooseVersion(self.SUPPORTED_FROM_VERSION):
+        from_version = input("\nThe fromversion value that will be used (optional): ")
+        if from_version:
+            yml_dict['fromversion'] = from_version
```

Comment:
please check if the input version is lower then `self.SUPPORTED_FROM_VERSION`. If so, let's bump it to the self.SUPPORTED_FROM_VERSION with a proper message.